<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>300</x>
    <y>300</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главное меню</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="start_button">
      <property name="text">
       <string>Начать игру</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="settings_button">
      <property name="text">
       <string>Настройки</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stats_button">
      <property name="text">
       <string>Статистика</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar"/>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
